README.md

```diff
@@ -0,0 +1,4 @@
+# th2 store common library (Java)
+
+This project contains the storage abstract class with boilerplate code to interact with the message router from the [th2-common](https://github.com/th2-net/th2-common-j).
+It also has got utility classes to convert th2's messages and events discribed in [th2-grpc-common](https://github.com/th2-net/th2-grpc-common/blob/master/src/main/proto/th2_grpc_common/common.proto "common.proto") to compatible with th2 storage format.
```

Comment:
It also **contains the** utility classes to convert th2's messages and **the** events **described** in [th2-grpc-common](https://github.com/th2-net/th2-grpc-common/blob/master/src/main/proto/th2_grpc_common/common.proto "common.proto") to **be** compatible with th2 storage format.

Reply:
Could you please tell, why should we write 'the' article before events and shouldn't write before messages?

Reply:
In this case I used the article 'the', because we are talking about specific events. Before messages, I omit the article 'the' because of the word that comes before - th2, although we can use 'the' as well because we are also talking about specific messages - both forms are correct

Reply:
Thanks